web_src/js/features/repo-issue.js

```diff
@@ -242,11 +247,20 @@ export function initRepoPullRequestMerge() {
       }
     }
   });
+  $('.merge-now-button > .dropdown').dropdown({
+    onChange(_text, _value, $choice) {
+      if ($choice.data('do')) {
```

Comment:
As discussed before, all static `data-` attribute should be accessed by `.attr('data-xxx')` to make the grep & search easier. https://docs.gitea.io/en-us/guidelines-frontend/#html-attributes-and-dataset